helm/cloudcore/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for cloudcore.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: kubeedge/cloudcore
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+serviceAccount:
+  # Annotations to add to the service account
+  annotations: {}
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  create: false
+  # type: NodePort
```

Comment:
We could set the default `service.type` to `ClusterIP`. This allows people to then choose whether they want to keep it internal or expose it through an ingress layer.

Reply:
Agreed here. NodePort can prove difficult to in practice in hosted versions of k8s backed by an elastic pool of nodes.